docs/crio.8.md

```diff
@@ -167,6 +168,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--enable-metrics**: Enable metrics endpoint for the server on localhost:9090
 
+**--enable-custom-shm-size**: Enable users to set custom shm size instead of default value 64M. The shmsize can be set through K8S annotation with key "io.kubernetes.cri-o.ShmSize", and the value representing the size in bytes.
```

Comment:
```suggestion
**--enable-custom-shm-size**: Enable users to set custom shm size instead of using the default value of 64M. The shm size can be set through K8S annotation with key "io.kubernetes.cri-o.ShmSize", and the value representing the size in bytes.
```